api/proto/info/info.proto

```diff
@@ -298,7 +301,6 @@ message DevicePort {
   ZInfoDNS dns = 25; // If DHCP assigned
   bool up = 26;      // operational up/down status.
   GeoLoc location = 27;
-  bool uplink = 28; // Uplink interface  // XXX rename to isMgmt
```

Comment:
Have you checked whether zedcloud references this field?

Reply:
This is new. Zedcloud doesn't use it